backend/box/box.go

```diff
@@ -384,16 +390,25 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 	}
 
 	root = parsePath(root)
-	oAuthClient, ts, err := oauthutil.NewClient(name, m, oauthConfig)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to configure Box")
+
+	boxAccessToken, boxAccessTokenOk := m.Get("box_primary_access_token")
+
+	client := &http.Client{}
```

Comment:
This should be `client := fshttp.NewClient(fs.Config)` so the default is to pick up rclone's http client, rather than the default http.Client. Using the default client means that `--tpslimit` and `--dump headers` etc won't work.